lib/ipmi_backend_utils.pm

```diff
@@ -13,7 +13,7 @@ use Exporter;
 use strict;
 use warnings;
 use testapi;
-use version_utils qw(is_storage_ng is_sle);
+use version_utils qw(is_storage_ng is_sle is_alp);
```

Comment:
this function is not found used in this file.

Reply:
yep... I forgot to clean it up after I moved changes out of it.

Reply:
updated